### PR TITLE
Diagram updates for 1.0 architecture changes

### PR DESCRIPTION
**Pull request by petercrocker**
Update diagrams for 1.0 release architecture changes & fix excalidraw svg issues

### CODE REVIEW

File: models/base_menu.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ spec:
           - namespace: Organization
             name: Manufacturer
             label: Manufacturer
-            kind: LocationManufacturer
```
Comment (dgarros) [inline models/base_menu.yml]: this has been fixed in 2 PRs already :)

File: docs/docs/media/high_level_architecture.excalidraw.svg
Comment (dgarros) [inline docs/docs/media/high_level_architecture.excalidraw.svg:1]: The task work still talk to the api server via the Graphql interface
I forgot to mention that earlier but the task worker also talks to the Graph DB, and the cache
also the arrow between the task manager and the task worker should be in the other direction

File: models/base_menu.yml (hunk 1 above)
Comment (petercrocker) [reply models/base_menu.yml]: 😂 I've reverted this so there's no conflicts.

File: docs/docs/media/high_level_architecture.excalidraw.svg
Comment (petercrocker) [reply docs/docs/media/high_level_architecture.excalidraw.svg:1]: Thanks, corrections done.

File: docs/docs/media/overview-interfaces.excalidraw.svg
Comment (dgarros) [inline docs/docs/media/overview-interfaces.excalidraw.svg:1]: Not sure why but I can't see these 2 files

**Review by dgarros — APPROVED** [review]
LGTM, minus the 2 files that I can't see

**Review by ogenstad — CHANGES_REQUESTED** [review]
As Damien indicated it's not possible to see two of the files in Github and the test build is also failing to show the images: https://e5bf92f6.infrahub.pages.dev/overview/

**Comment by petercrocker** [comment]
> As Damien indicated it's not possible to see two of the files in Github and the test build is also failing to show the images: https://e5bf92f6.infrahub.pages.dev/overview/

Strange. No idea why. I re-saved the files, works now in cloudflare tests: https://pmc-20241024-svgs.infrahub.pages.dev/overview/

**Comment by ogenstad** [comment]
> Strange. No idea why. I re-saved the files, works now in cloudflare tests: https://pmc-20241024-svgs.infrahub.pages.dev/overview/

Solar flares..

**Pull request merged**